src/components/EmployeesList/EmployeesList.tsx

```diff
@@ -38,58 +34,64 @@ const EmployeesList: React.FC<EmployeesListProps> = ({
 
   const handleDeleteButtonClick = (record: Employee) => {
     setIsDeleteModalOpen(true)
-    setNewEmployeeContent(record)
+    setChosenEmployee(record)
     setExpandedRowKeys([])
   }
 
   const handleUpdateButtonClick = (record: Employee) => {
-    setIsNewEmployeeModalOpen(true)
-    setNewEmployeeContent(record)
+    setIsUpdateEmployeeModalOpen(true)
+    setChosenEmployee(record)
     setExpandedRowKeys([])
   }
 
   const handleExpand = (expanded: boolean, record: Employee) => {
-    setNewEmployeeContent(record)
+    setChosenEmployee(record)
     setExpandedRowKeys((prevRowKeys) => {
-      if (prevRowKeys[0] === record.key) {
+      if (prevRowKeys[0] === record.id) {
         return []
       }
-      return [record.key]
+      return [record.id]
     })
   }
 
-  const expandableConfig = {
+  const expandableConfig: ExpandableConfig<Employee> = {
     expandRowByClick: true,
     expandedRowRender: (record: Employee) => {
       return (
         <ExpandedRow>
-          <Button type='link' onClick={() => navigate(`${location.pathname}/${record.key}`)}>
+          <Button type='link' onClick={() => navigate(`${location.pathname}/${record.id}`)}>
             View Profile
           </Button>
-          <UpdateButton type='default' onClick={() => handleUpdateButtonClick(record)}>
-            Update
-          </UpdateButton>
-          <Button type='primary' onClick={() => handleDeleteButtonClick(record)}>
-            Delete
-          </Button>
+          {isAdmin && (
+            <>
+              <UpdateButton type='default' onClick={() => handleUpdateButtonClick(record)}>
+                Update
+              </UpdateButton>
+              <Button type='primary' onClick={() => handleDeleteButtonClick(record)}>
+                Delete
+              </Button>
+            </>
+          )}
         </ExpandedRow>
       )
     }
   }
 
-  const paginationConfig = {
-    pageSize: 6
-  }
-
   return (
     <Table
-      pagination={paginationConfig}
+      tableLayout='auto'
+      pagination={false}
       expandedRowKeys={expandedRowKeys}
       onExpand={handleExpand}
       expandable={expandableConfig}
       dataSource={employeeList}
       columns={columns}
-      size='large'
+      bordered
+      loading={isFetching}
+      rowKey='id'
+      scroll={{
+        y: 400
```

Comment:
what is 400

Reply:
This value controls the height of antd table. AFAIK, there is no other way to control it